sites/platform/src/create-apps/app-reference.md

```diff
@@ -193,50 +193,152 @@ For more information, see how to [manage resources](/manage-resources.md).
-![Relationships Diagram](/images/management-console/relationships.png "0.5")
-{{% /version/only %}}
+The quickest way to define a relationship between your app and a service
+is to use the service container's default endpoint.</br>
+However, some services allow you to define multiple databases, cores, and/or permissions.
+In these cases, you can't rely on default endpoints.
+Instead, you can explicitly define multiple endpoints when setting up your relationships.
 
-You can give each relationship any name you want.
-This name is used in the `PLATFORM_RELATIONSHIPS` environment variable,
-which gives you credentials for accessing the service.
+{{< note >}}
+App containers don't have a default endpoint like service containers.
+To connect your app to another app in your project,
+explicitly define the `http` endpoint as the endpoint to connect both apps.</br>
+For more information, see how to [define relationships between your apps](/create-apps/multi-app/relationships.md).
+{{< /note >}}
 
-The relationship is specified in the form `service_name:endpoint_name`.
-The `service_name` is the name of the service given in the [services configuration](../add-services/_index.md)
-or the name of another application in the same project specified as the `name` in that app's configration.
+To define a relationship between your app and a service:
 
-The `endpoint_name` is the exposed functionality of the service to use.
-For most services, the endpoint is the same as the service type.
-For some services (such as [MariaDB](../add-services/mysql/_index.md#multiple-databases) and [Solr](../add-services/solr.md#solr-6-and-later)),
-you can define additional explicit endpoints for multiple databases and cores in the [service's configuration](../add-services/_index.md).
+{{< codetabs >}}
 
-The following example shows a single MySQL service named `mysqldb` offering two databases,
-a Redis cache service named `rediscache`, and an Elasticsearch service named `searchserver`.
++++
+title=Using default endpoints
++++
+
+Use the following configuration:
+
+```yaml {configFile="app"}
+{{< snippet>}}
+relationships:
+    {{% variable "SERVICE_NAME" %}}: 
+{{< /snippet >}}
+```
+
+The `SERVICE_NAME` is the name of the service as defined in its [configuration](/add-services/_index.md).
+It is used as the relationship name, and associated with a null value.
+This instructs {{% vendor/name %}} to use the service container's default endpoint to connect your app to the service.
+
+For example, if you define the following configuration:
 
-{{% version/specific %}}
 ```yaml {configFile="app"}
+{{< snippet>}}
 relationships:
-    database: 'mysqldb:db1'
-    database2: 'mysqldb:db2'
-    cache: 'rediscache:redis'
-    search: 'searchserver:elasticsearch'
+    mariadb: 
+{{< /snippet>}}
 ```
+
+{{% vendor/name %}} looks for a service named `mariadb` in your `{{% vendor/configfile "app" %}}` file,
+and connects your app to it through the service container's default endpoint.
```

Comment:
It might worth showing the equivalent with the explicit notation:

```yaml
relationships
  mariadb:
    service: mariadb
    endpoint: mysql
```